plugin/src/test/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildActionTest.java

```diff
@@ -88,4 +120,46 @@ void shouldCreateViewModel() {
         assertThat(action.getTarget()).extracting(CoverageViewModel::getNode).isSameAs(root);
         assertThat(action.getTarget()).extracting(CoverageViewModel::getOwner).isSameAs(action.getOwner());
     }
+
+    @Test
+    void shouldReturnPositiveTrendForLineMetric() {
+        CoverageBuildAction action = createCoverageBuildActionWithDelta();
+        double trend = action.getTrend(Baseline.PROJECT, Metric.LINE);
+        assertThat(trend).isEqualTo(0.3);    // deltaValue = 0.3
```

Comment:
Almost: what I actually meant is something like:
```suggestion
        CoverageBuildAction action = createCoverageBuildActionWithDelta(Baseline.PROJECT, Metric.LINE, Fraction.of("0.001");
        assertThat(action.getTrend(Baseline.PROJECT, Metric.LINE)).isPositive();
```

and then 

```
        CoverageBuildAction action = createCoverageBuildActionWithDelta(Baseline.PROJECT, Metric.LINE, Fraction.of("0.0009");
        assertThat(action.getTrend(Baseline.PROJECT, Metric.LINE)).isZero();
```

So move the interesting parts to the focus of the reader.

Reply:
Maybe it would be helpful to add here: 
```
assertThat(action.formatDelta(Baseline.PROJECT, Metric.LINE)).isEqualTo("+0.10%");
```